### DISCUSSION TITLE
Sensitivity Analysis

### DISCUSSION THREAD

**Discussion by darrenyu1111** [coin-or/pulp#467] [Q&A]
Hello,
 
   Sensitivity analysis in pulp, do we have more attributes besides Shadow Price and slack, for example, like in excel, constraint R.H. Side,  allowable Increase, allowable decrease? thanks.
 
  	       
best regards,      		
darren yu

**Comment by pchtsp** [comment]
Hello Darren,

I think not. Although you can calculate that from the constraints and the values of the variables if you want to.
